src/main/scala/codacy/plugins/test/MultipleTests.scala

```diff
@@ -0,0 +1,143 @@
+package codacy.plugins.test
+
+import java.nio.file.Path
+
+import codacy.utils.Printer
+import com.codacy.analysis.core.model.{
+  CodacyCfg,
+  Configuration,
+  FileCfg,
+  FileError,
+  FullLocation,
+  Issue,
+  LineLocation,
+  Parameter,
+  Pattern
+}
+import com.codacy.analysis.core.tools.Tool
+import com.codacy.plugins.api.results.Result
+import com.codacy.plugins.api.results.Result.Level
+import com.codacy.plugins.results.traits.{DockerToolDocumentation, ToolRunner}
+import com.codacy.plugins.runners.{BinaryDockerRunner, DockerRunner}
+import com.codacy.plugins.utils.BinaryDockerHelper
+import com.codacy.plugins.results.PluginResult
+
+import better.files._
+
+import play.api.libs.json.Json
+
+import scala.util.{Failure, Success, Try}
+import com.fasterxml.jackson.core.JsonParseException
+import scala.xml.XML
+
+object MultipleTests extends ITest with CustomMatchers {
+
+  val opt = "multiple"
+
+  def run(testDirectories: Seq[Path], dockerImage: DockerImage, optArgs: Seq[String]): Boolean = {
+    Printer.green(s"Running MultipleTests:")
+    val testSources = testDirectories.filter(_.getFileName.toString == DockerHelpers.multipleTestsDirectoryName)
+
+    val languages = findLanguages(testSources, dockerImage)
+    val dockerTool = createDockerTool(languages, dockerImage)
+    val toolDocumentation = new DockerToolDocumentation(dockerTool, new BinaryDockerHelper(useCachedDocs = false))
+    val dockerRunner = new BinaryDockerRunner[Result](dockerTool)()
+    val runner = new ToolRunner(dockerTool, toolDocumentation, dockerRunner)
+    val tools = languages.map(new Tool(runner, DockerRunner.defaultRunTimeout)(dockerTool, _))
+    testSources.forall { multipleTestsDir =>
+      val testsDirectories = File(multipleTestsDir).list
```

Comment:
`testsDirectories` name clashes with the argument of the method making it difficult to reason about, would you mind to change one of the two?